playbooks/cloud-provider-openstack-multinode-csi-migration-test/pre.yaml

```diff
@@ -209,16 +209,20 @@
           set -o pipefail
           set -ex
           # Specify target architecture to build binaries and images
-          export ARCH=${ARCH:-amd64}
-          make build-cmd-cinder-csi-plugin
-          docker build -t {{ registry_ip }}:32000/k8scloudprovider/cinder-csi-plugin:latest -f cluster/images/cinder-csi-plugin/Dockerfile.build .
+          export ARCHS=${ARCHS:-amd64}
+          export VERSION=${VERSION:-latest}
+          export REGISTRY=${REGISTRY:-{{ registry_ip }}:32000/k8scloudprovider}
+          export BUILD_CMDS=cinder-csi-plugin
+          export IMAGE_NAMES=cinder-csi-plugin
 
-    - name: push cluster-local-build of cinder-csi-plugin to local registry
-      shell:
-        cmd: |
-          docker push {{ registry_ip }}:32000/k8scloudprovider/cinder-csi-plugin:latest
+          # Docker login information here and no_log tag below will be removed once
+          # https://github.com/kubernetes/cloud-provider-openstack/pull/967 landed
+          export DOCKER_PASSWORD='{{ dockerhub.password }}'
```

Comment:
its a local registry server , these credentials are not required

Reply:
Unfortunately, it's required due to my pool design, which will be correct in [1]

[1] https://github.com/kubernetes/cloud-provider-openstack/pull/967/commits/323dbe9a4347bdbcf95209ec744ad307306dbe17#diff-b67911656ef5d18c4ae36cb6741b7965R271